src/dmd/backend/cod4.c

```diff
@@ -2062,16 +2067,6 @@ void cdcmp(CodeBuilder& cdb,elem *e,regm_t *pretregs)
                     code_orflag(cdb.last(), CFpsw);
                     cdb.genc2(0x81,grex | modregrmx(3,2,reg),0);          // ADC reg,0
                     goto oplt;
-                case OPgt:
-                    cdb.gen2(0xF7,grex | modregrmx(3,3,reg));         // NEG reg
-#if TARGET_WINDOS
-                    // What does the Windows platform do?
-                    //  lower INT_MIN by 1?   See test exe9.c
-                    // BUG: fix later
-                    code_orflag(cdb.last(), CFpsw);
-                    cdb.genc2(0x81,grex | modregrmx(3,3,reg),0);  // SBB reg,0
```

Comment:
I don't know why this section was #if'd only for Windows. The SBB instruction makes it work. No need to disable the optimization.